osu.Game/Overlays/Notifications/Notification.cs

```diff
@@ -38,6 +38,11 @@ public abstract partial class Notification : Container
         /// </summary>
         public virtual bool IsImportant => true;
 
+        /// <summary>
+        /// Whether this notification should trigger a taskbar flash if the window is un-focused when posted.
+        /// </summary>
+        public bool FlashTaskbar { get; init; } = true;
```

Comment:
I dispute this being on by default. There are many more places that probably shouldn't have that.

The framework error-to-notification interceptor:

https://github.com/ppy/osu/blob/b581d67416d569a14b29f6a969c4ea922729c803/osu.Game/OsuGame.cs#L1129-L1161

And probably most of these:

https://github.com/ppy/osu/blob/b581d67416d569a14b29f6a969c4ea922729c803/osu.Desktop/Security/ElevatedPrivilegesChecker.cs#L37
https://github.com/ppy/osu/blob/b581d67416d569a14b29f6a969c4ea922729c803/osu.Game/Overlays/BeatmapSet/Buttons/FavouriteButton.cs#L82-L86

I'm not sure what https://github.com/ppy/osu/pull/24588#discussion_r1300870478 intended to say, but if it intended to say "important notifications should flash and unimportant should not", then that's not what's happening here right now. What _is_ happening right now is "important notifications perma-flash and unimportant flash once unless explicitly turned off".

Reply:
> but if it intended to say "important notifications should flash and unimportant should not", then that's not what's happening here right now

that's what i intended to propose, not adding any new variable (more variable = more confusing).

Reply:
I've just nuked `FlashTaskbar` (https://github.com/ppy/osu/pull/24588/commits/aa29e00578a01768d1ee7bf537e5b2a335c46b30) and switched to use `IsImportant` 100% instead. It still isn't all there where I'd expect to be UX-expectation-wise but it's probably fine until proven otherwise.

Notably I had to apply https://github.com/ppy/osu/pull/24588/commits/142abe1fd0a87a2144ee4f61c40cff1c96ce4605 too to not defeat the original purpose of this PR (which was to make highlights/PMs flash).